utils/xlsx_reader.py

```diff
@@ -37,7 +38,14 @@ def create_library_item(session, model, **kwargs):
     if library_item:
         return library_item
     else:
-        instance = model(**kwargs)
+        language = choice(['polish', 'other', 'english'])
+        if model.__name__ == "Book":
```

Comment:
this if-else statement for randomly assigning language and pub_date should be only available for testing and dev purposes since the production .xlsx file has this data
some switch-case/flag is needed in order to handle this difference